README.md

```diff
@@ -6,8 +6,10 @@ merchandise, etc. for all the gymkhana clubs in our institute.
 
 
 ## Tech stack
-- ui - React(Typescript)
-- api - Django
+- **FrontEnd:** Single Page Application using [React](https://reactjs.org/)([TypeScript](https://www.typescriptlang.org/))
+- **API:** [Django Rest Framework](https://www.django-rest-framework.org/) for the API ([Python](https://www.python.org/))
+- **PullRequest validation:** [Github Actions](https://github.com/features/actions) to do automated PR validation by running tests and linter
+- **Version Control and Hosting:** [Git](https://git-scm.com/) for Source Code Management and [Github](https://github.com) for hosting
```

Comment:
Is this necessary? 
I think it is obvious that we are using git & Github.

Reply:
I don't mind being verbose about the technology we are using, if it doesn't hurt developers.